app/scripts/modules/core/pipeline/config/stages/executionWindows/ExecutionWindowDayPickerComponent.js

```diff
@@ -0,0 +1,89 @@
+import angular from 'angular';
+
+class ExecutionWindowDayPickerController {
+
+  constructor() {
+    this.DAYS = [{
```

Comment:
It's a little confusing the have both `DAYS` and `days` on the controller. Maybe change this to `dayOptions`?